x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/DenseVectorFieldMapper.java

```diff
@@ -494,6 +495,13 @@ protected String contentType() {
         return new Builder(simpleName(), indexCreatedVersion).init(this);
     }
 
+    @Override
+    public void doValidate(MappingLookup mappers) {
```

Comment:
Do we want in the future to add a support for vector fields under nested? Perhaps when `_knn_search`  becomes a a part of `_search`?

Reply:
Yes, we're definitely open to it as a follow-up. I'd like to better understand the use cases for nested before introducing it.